#Request

float64[] final_position
bool send

---

#Response

bool response
